src/main/java/explorableviz/transparenttext/Main.java

```diff
@@ -112,4 +130,53 @@ public static Map<String, String> parseArguments(String[] args) {
                         (_, replacement) -> replacement
                 ));
     }
+
+    public static void generateLinks() throws Exception {
```

Comment:
Is it not possible to take a similar approach to the one you used for the webpages themselves, and just put a substitution token in the template that you then replace with a list of `<a href=""></a>` elements. But if this works, let's go with this and we can discuss on Monday!